.github/workflows/php-test-auto-sizes-plugin.yml

```diff
@@ -0,0 +1,78 @@
+name: Unit Testing for Auto-sizes for Lazy-Loaded Images Plugin
```

Comment:
We'll end up with _a lot_ of jobs if we have a separate workflow for each plugin, and then each workflow running a matrix of like 10 items.

Should we just have 1 file for running all these test suites?

Reply:
In https://github.com/WordPress/performance/pull/972/commits/88040f2ed751b6ecb8d9108e2c471759e36e29e3 added single file.